lucene/ivy-versions.properties

```diff
@@ -36,7 +34,7 @@ com.google.inject.guice.version = 3.0
 /com.google.inject.extensions/guice-servlet = ${com.google.inject.guice.version}
 /com.google.inject/guice = ${com.google.inject.guice.version}
 
-/com.google.protobuf/protobuf-java = 2.5.0
+/com.google.protobuf/protobuf-java = 3.1.0
```

Comment:
@markrmiller Can this be updated with the Hadoop support? Tests seem to pass but not sure if this affects protobuf compatibility with a running cluster.